### DISCUSSION TITLE
Multiline item description indentation

### DISCUSSION THREAD

**Discussion by SevenOutman** [jotaen/xit#17] [File Format]
Hi Jan, thanks for making this specification!

I have some questions about indentation in multiline item descriptions. [Quoting the specification](https://github.com/jotaen/xit/blob/aae36c034cc139f742b14e1881a05a3ad039966f/Specification.md?plain=1#L96), the subsequent lines

> MAY be continued on the subsequent line(s),
where each of them MUST be indented (preceded) by a sequence of four space characters (`    `)
> #### Example
>
> ```
> [ ] This description is one line
> [ ] This description continues ...
>     ... on the next line
> ```

What about items with priorities specified? Their descriptions don't start at the 5th column, must they also have exactly 4 spaces before subsequent lines?

**Comment by jotaen** [comment]
I would say so, yes.

```
[ ] This description is one line
[ ] This description continues ...
    ... on the next line
[ ] !!! This is important, and it also ...
    continues on the next line
```

I think otherwise the indentation rules might become too complicated, especially when [nested subtasks](https://github.com/jotaen/xit/discussions/2) would be supported.

It would probably be good to add in some more examples to the spec, to demonstrate such cases.
